ui/ui-components/components/record/SampleRecordCard.tsx

```diff
@@ -334,50 +384,60 @@ const SampleRecordCard: React.FC<SampleRecordCardProps> = ({
         {email && (
           <RecordImageFromEmail email={email} width={72} className="mb-4" />
         )}
-        <h6>Groups</h6>
-        {
-          <p>
-            {groups?.length
-              ? groups.map((group) => {
-                  return (
-                    <Fragment key={`group-${group.id}`}>
-                      <Link
-                        href={`/model/${group.modelId}/group/${group.id}/${
-                          group.type === "calculated" ? "rules" : "edit"
-                        }`}
-                      >
-                        {group.name}
-                      </Link>
-                      <br />
-                    </Fragment>
-                  );
-                })
-              : "None"}
-          </p>
-        }
-        <h6>Destinations</h6>
-        {
-          <p>
-            {destinations?.length
-              ? destinations.map((destination) => {
-                  return (
-                    <Fragment key={`destination-${destination.id}`}>
-                      {isConfigUI ? (
-                        destination.name
-                      ) : (
-                        <EnterpriseLink
-                          href={`/model/${destination.modelId}/destination/${destination.id}/data`}
-                        >
-                          <a>{destination.name}</a>
-                        </EnterpriseLink>
-                      )}
-                      <br />
-                    </Fragment>
-                  );
-                })
-              : "None"}
-          </p>
-        }
+        <h6>{groupsTitle}</h6>
```

Comment:
Nice - I really like how reusable you made this component.  It's gnarly, but it's very powerful.